app/views/user_mailer/invite_request_declined.text.erb

```diff
@@ -1,9 +1,9 @@
 <% content_for :message do %>
-<%= ts("Dear #{@user.login}, ")%>
-<%= ts("We regret to inform you that your request for #{@total} new ") %><%= @total == 1 ? ts("invitation") : ts("invitations") %><%= ts(" cannot be fulfilled at this time.") %>
-<%= ts("Your request was: ") %>
+<%= t('.greeting', user: @user.login) %>
+<%= t('.main', count: @total) %>
+<%= t('.reason') %>
 <%= to_plain_text(raw @reason) %>
 
-Regards,
-AO3
+<%= t '.signoff' %>
```

Comment:
Can you add the parentheses to this for consistency? Or does it need to be done without parentheses?

Reply:
Yes, fine.